src/appMain/CMakeLists.txt

```diff
@@ -76,26 +63,26 @@ endif()
 
 set(LIBRARIES
   MediaManager
-  ApplicationManager
+  application_manager
   HMI_API
   MOBILE_API
   v4_protocol_v1_2_no_extra
-  SmartObjects
+  smart_object
   ${TELEMETRY_MONITOR_LIB}
   formatters
-  ProtocolLibrary
-  ProtocolHandler
-  connectionHandler
+  protocol
+  protocol_handler
+  connection_handler
   TransportManager
   ${SecurityManagerLibrary}
   HMIMessageHandler
   MessageBroker
   ${BROKER_LIBRARIES}
   Utils
   jsoncpp
-  ConfigProfile
-  Resumption
-  UsageStatistics
+  config_profile
+  resumption
+  usage_statistics
 )
```

Comment:
@anosach-luxoft 
Maybe it's worth to have same naming convention for all libraries?